routes/web.php

```diff
@@ -69,3 +70,11 @@
 
     return redirect()->route('login')->with('success', 'Logout berhasil!');
 })->name('logout');
+
+Route::get('/ayats',AyatController::class .'@index')->name('Ayats/ayats');
+Route::post('/ayats/posts', AyatController::class .'@store')->name('ayats.store');
+Route::get('/ayats/posts/create', AyatController::class .'@create')->name('ayats.create');
+Route::get('/ayats/posts/{post}', AyatController::class .'@show')->name('ayats.show');
+Route::get('/ayats/posts/{post}/edit', AyatController::class .'@edit')->name('ayats.edit');
+Route::put('/ayats/posts/{post}', AyatController::class .'@update')->name('ayats.update');
+Route::delete('/ayats/posts/{post}', AyatController::class .'@destroy')->name('ayats.destroy');
```

Comment:
Seharusnya membuat books terlebih dahulu untuk routing nya bisa nambahin parameter contoh isi dari tables:

|id|books|slug|api_books|
| ---| --- |---| ---|
|1|al-quran|al-quran|equran.id/api/v2|
|2|bibble|bibble-katolik|api.bible.com/api/v1|

Example:
1. ```12.7.0.0.1/slugs/chapter/ayats```
atau bisa pakai parameter 
2. ```127.0.0.1/ayats?books=slug&chapters=chapter```

Output yang diharapkan: menampilkan semua ayat pada chapters tersebut.

Untuk tables ayats ini mungkin bisa dijadikan store content favorite nanti bisa melihat ayat favorit serta isi content favorit tersebut tanpa membuka chapter

Note: Belum bisa merge to main sebelum perbaiki model dari books, dan membuat method pada tabel yang bersangkutan.